Makefile

```diff
@@ -223,10 +223,14 @@ test-ui-e2e:
 test-kitchensink-e2e-testonly:
 	./test/kitchensink-e2e-tests.sh
 
-test-kitchensink-e2e:
+test-kitchensink-e2e-single-testonly:
+	./test/kitchensink-e2e-tests.sh -run $(TEST)
+
+test-kitchensink-e2e-setup:
 	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
 	SCALE_UP=4 INSTALL_KAFKA="true" ./hack/install.sh
-	./test/kitchensink-e2e-tests.sh
+
+test-kitchensink-e2e: test-kitchensink-e2e-setup test-kitchensink-e2e-testonly
```

Comment:
Unfortunately, it's not guaranteed that `test-kitchensink-e2e-setup` will run before `test-kitchensink-e2e-testonly`. That burned me in the past. It's better to list the Bash functions under `test-kitchensink-e2e`, one by one.

Reply:
OK